CMakeLists.txt

```
@@ -183,6 +184,18 @@ else()
   find_package(SDL2_mixer REQUIRED)
 endif()
 
+include(FetchContent)
+FetchContent_Declare(
+  spdlog
+  GIT_REPOSITORY https://github.com/gabime/spdlog.git
+  GIT_TAG        100f30043f33277122e0991c83845a2617172ffd
+)
+FetchContent_GetProperties(spdlog)
+if(NOT spdlog_POPULATED)
+  FetchContent_Populate(spdlog)
+  add_subdirectory(${spdlog_SOURCE_DIR} ${spdlog_BINARY_DIR})
+endif()

```

Use our `FetchContent_MakeAvailableExcludeFromAll` helper:

```suggestion
include(FetchContent_MakeAvailableExcludeFromAll)
FetchContent_MakeAvailableExcludeFromAll(spdlog)
```